### PR TITLE
Site cleanup

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

* Move files in assets/img to assets/images and replace with a symlink
* Make logo image relative
* Update README.md

### CODE REVIEW

**Comment by soulshake** [comment]
[Site README tweaks](https://trello.com/c/lGuUswUw/1517-site-readme-tweaks)

File: README.md [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-Pull Requests are also welcome!
+Pull requests are also welcome!
 
 ## Usage
 
+Convox and Docker are recommended.
+
+```shell
+$ git clone https://github.com/convox/site.git && cd site
+$ echo DEVELOPMENT >> .env
```
Comment (mwarkentin) [inline README.md]: This doesn't look quite right? I had to set `DEVELOPMENT` to a value, and also `JEKYLL_ENV` if I recall correctly?

Comment (soulshake) [inline README.md]: Ha whoops thanks
`JEKYLL_ENV` gets set [here](https://github.com/convox/site/blob/master/_bin/web#L8), btw.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+Pull requests are also welcome!
 
 ## Usage
 
+Convox and Docker are recommended.
+
+```shell
+$ git clone https://github.com/convox/site.git && cd site
+$ echo DEVELOPMENT=true >> .env
+$ convox start
```
Comment (mattmanning) [inline README.md]: Also need `JEKYLL_ENV=development` here, I think.

Comment (soulshake) [inline README.md]: `JEKYLL_ENV` gets set [here](https://github.com/convox/site/blob/master/_bin/web#L8).

Comment (mattmanning) [inline README.md]: Ah thanks missed that.

**Pull request merged**